packages/contracts/src/core/permission/PermissionLib.sol

```diff
@@ -6,18 +6,35 @@ pragma solidity 0.8.17;
 /// @author Aragon Association - 2021-2023
 /// @notice A library containing objects for permission processing.
 library PermissionLib {
+    /// @notice A constant expressing that not condition is applied to a permission.
+    address constant NO_CONDITION = address(0);
```

Comment:
Doesn't this give a warning ? maybe ambiguity and better add `public` or I completely forgot if you could add `public` at all in the library. Just mentioning. You decide.

Reply:
Added public a8e74aca1abbbb21bc7fb5ce3cff9d7aca8a83b1